forte/data/base_pack.py

```
@@ -193,10 +195,43 @@ def _deserialize(
 
         if serialize_method == "jsonpickle":
             with _open(data_source, mode="rt") as f:  # type: ignore
-                pack = cls.from_string(f.read())
+                try:
+                    pack = cls.from_string(f.read())
+                except AttributeError as e:
+                    logging.exception(
+                        "There was an unknown type or attribute during"
+                        "deserialization and broke the process."
+                        "It might be caused by adding"
+                        "new classes or new attributes to the ontology."
+                        "Details are: %s",
+                        e,
+                    )
+                    # raise AttributeError(
+                    #     "There was an unknown type or attribute during"
+                    #     "deserialization and broke the process."
+                    #     "It might be caused by adding"
+                    #     "new classes or new attributes to the ontology."
+                    # ) from e
+
         else:
             with _open(data_source, mode="rb") as f:  # type: ignore
-                pack = pickle.load(f)
+                try:
+                    pack = pickle.load(f)
+                except AttributeError as e:
+                    logging.exception(
+                        "There was an unknown type or attribute during"
+                        "deserialization and broke the process."
+                        "It might be caused by adding"
+                        "new classes or new attributes to the ontology."
+                        "Details are: %s",
+                        e,
+                    )
+                    # raise AttributeError(
```

I think we should raise the exception here and "fail fast"